astropy/io/fits/scripts/fitsdiff.py

```diff
@@ -80,7 +80,8 @@ def __call__(self, parser, namespace, values, option_string=None):
                 log.warning(f"{self.dest} argument {value} does not exist")
                 return
             try:
-                values = [v.strip() for v in open(value).readlines()]
+                with open(value) as fd:
+                    values = fd.read().splitlines()
```

Comment:
If this must be changed, please just change this to `values = [v.strip() for v in fd]`. It is NOT worth the out time to determine if the `strip()` is really needed. The original developer wrote it this way for a reason!

Reply:
Agreed. So #16630 is sufficient here. I'll revert this one too